Dockerfile-ubuntu

```diff
@@ -16,3 +16,21 @@ RUN git clone https://github.com/hybridgroup/gocv.git
 WORKDIR gocv
 RUN make deps
 RUN make install
+
+# Install larynx
+RUN pip3 install --upgrade pip
+RUN pip3 install --upgrade wheel setuptools regex==2022.3.2
+RUN pip3 install -f 'https://synesthesiam.github.io/prebuilt-apps/' -f 'https://download.pytorch.org/whl/cpu/torch_stable.html' larynx==1.1.0
+
+# Build oor
+FROM build AS ocr-container
+ENV CGO_ENABLED 1
+ENV GOOS "linux"
+ENV GOARCH "amd64"
+ARG VERSION
+ARG BUILD_ARGS
+ADD . /open-ocr-reader/
+WORKDIR /open-ocr-reader/
+RUN apt install -y tesseract-ocr-eng
+RUN go build ${BUILD_ARGS} -ldflags "-X github.com/epinio/epinio/internal/version.Version=${VERSION}" -o dist/oor-linux-amd64
+ENTRYPOINT ["dist/oor-linux-amd64", "server"]
```

Comment:
This block is not needed. Check how it's used here: https://github.com/jimmykarily/open-ocr-reader/blob/2a66b6c23c5274ee21e60f770053d94c8795c83e/Makefile#L12

We can add another target to start/stop the server (or just run it in the foreground). This image is just a helper for development so that we can build and run the binary in a stable environment. We can have another image in the future that can be used for deployment which will have just the built binary + its runtime dependencies.